client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-intro/intro.tsx

```diff
@@ -46,6 +46,7 @@ const Intro: React.FC< Props > = ( { onSubmit } ) => {
 					{
 						key: 'finalize',
 						title: __( 'Checkout' ),
+						badge: 'Free',
```

Comment:
Should the badge text be translated?
```suggestion
						badge: __( 'Free' ),
```